tools/mo/openvino/tools/mo/front/tf/AutomlEfficientDet.py

```diff
@@ -48,7 +48,6 @@ def get(self, layer_id):
     def transform_graph(self, graph: Graph, replacement_descriptions: dict):
         parameter_node = graph.get_op_nodes(op='Parameter')[0]
         parameter_node['data_type'] = data_type_str_to_np(parameter_node.graph.graph['cmd_params'].data_type)
-        parameter_node.out_port(0).disconnect()
```

Comment:
Now it looks that `preprocessing_input_node` is not removed from the graph because `parameter_node` stills connects it but `preprocessing_input_node` does not have output node. Is the graph consistent and do the model conversion and inference still work?

Reply:
In 63-64 lines we get next node after preprocessing_input_node and set its source to Parameter node. So preprocessing_input_node has no output node at this point and it is removed during clean up. So resulting graph is the same. 
I have checked inference result, it is the same as on master.

Reply:
We can add disconnect after set_source() on 64 line to be sure that preprocessing_input_node is removed, but it is not necessary.